packages/fx-core/src/plugins/resource/aad/index.ts

```diff
@@ -90,7 +90,7 @@ export class AadAppForTeamsPlugin implements Plugin {
         ctx.logProvider?.error(`Detailed error: ${e.innerError.message}`);
       }
       TelemetryUtils.init(ctx);
-      TelemetryUtils.sendException(stage, e.name, e instanceof UserError ? Telemetry.userError : Telemetry.systemError, e.message );
+      TelemetryUtils.sendErrorEvent(stage, e.name, e instanceof UserError ? Telemetry.userError : Telemetry.systemError, e.message );
```

Comment:
errorCode format should be ${e.source}.${e.name}

Reply:
Fixed. Thanks!